src/jumio-kyc/kyc.controller.ts

```diff
@@ -80,4 +81,41 @@ export class KycController {
 
     return serializeKyc(redemption, jumioTransaction);
   }
+
+  @ApiExcludeEndpoint()
+  @Get('config')
+  config(): SerializedKycConfig {
+    return {
+      data: [
+        {
+          airdrop_completed_by: new Date(Date.UTC(2023, 2, 13, 0, 0, 0)),
+          coins: 105000,
+          kyc_completed_by: new Date(Date.UTC(2023, 2, 16, 0, 0, 0)),
+          name: 'Pull Requests',
+          pool_name: 'Code Contributions Pool',
+        },
+        {
+          airdrop_completed_by: new Date(Date.UTC(2023, 2, 19, 0, 0, 0)),
+          coins: 420000,
+          kyc_completed_by: new Date(Date.UTC(2023, 2, 23, 0, 0, 0)),
+          name: 'Phase 1',
+          pool_name: 'Phase 1 Pool',
+        },
+        {
+          airdrop_completed_by: new Date(Date.UTC(2023, 2, 26, 0, 0, 0)),
+          coins: 210000,
+          kyc_completed_by: new Date(Date.UTC(2023, 2, 30, 0, 0, 0)),
+          name: 'Phase 2',
+          pool_name: 'Phase 2 Pool',
+        },
+        {
+          airdrop_completed_by: new Date(Date.UTC(2023, 2, 26, 0, 0, 0)),
+          coins: 210000,
+          kyc_completed_by: new Date(Date.UTC(2023, 3, 6, 0, 0, 0)),
+          name: 'Phase 3',
+          pool_name: 'Phase 3 Pool',
+        },
+      ],
```

Comment:
Should we key this by pool? I think it'll be easier for dan since he needs to reference them explicitly in his design rather than loop over them.

 - pool1: {}
 - pool2: {}
 - pool3: {}
 - pool4: {}